public/app/features/plugins/admin/components/PluginDetailsPage.tsx

```diff
@@ -73,26 +74,34 @@ export function PluginDetailsPage({
     );
   }
 
+  const conditionalProps = !config.featureToggles.pluginsDetailsRightPanel ? { info: info } : {};
+  const widthPage = !config.featureToggles.pluginsDetailsRightPanel ? '100%' : '80%';
+
   return (
-    <Page navId={navId} pageNav={navModel} actions={actions} subTitle={subtitle} info={info}>
-      <Page.Contents>
-        <TabContent className={styles.tabContent}>
-          {plugin.angularDetected && (
-            <AngularDeprecationPluginNotice
-              className={styles.alert}
-              angularSupportEnabled={config?.angularSupportEnabled}
-              pluginId={plugin.id}
-              pluginType={plugin.type}
-              showPluginDetailsLink={false}
-              interactionElementId="plugin-details-page"
-            />
-          )}
-          <PluginDetailsSignature plugin={plugin} className={styles.alert} />
-          <PluginDetailsDisabledError plugin={plugin} className={styles.alert} />
-          <PluginDetailsDeprecatedWarning plugin={plugin} className={styles.alert} />
-          <PluginDetailsBody queryParams={Object.fromEntries(queryParams)} plugin={plugin} pageId={activePageId} />
-        </TabContent>
-      </Page.Contents>
+    <Page navId={navId} pageNav={navModel} actions={actions} subTitle={subtitle} {...conditionalProps}>
+      <Stack gap={4} justifyContent="space-between" direction={{ xs: 'column-reverse', sm: 'row' }}>
+        <div style={{ display: 'block', width: widthPage }}>
```

Comment:
did you consider instead using the grid component from our design system? 
I am not 100% sure it works on this case but worth to try a responsive solution
 https://developers.grafana.com/ui/latest/index.html?path=/docs/general-layout-grid--docs